### DISCUSSION TITLE
A very suitable place!

### DISCUSSION THREAD

**Discussion by wc-yaa** [VBAndCs/sVB-Small-Visual-Basic#89] [General]
https://litdev.uk/mybb/forumdisplay.php?fid=5

The above place is quite active. You can really show them how to solve each question in Small Visual Basic. Yes, show them the advantages of Small Visual Basic. In conclusion, it should be a nice place for the effective promotion of Small Visual Basic. Right?

**Comment by VBAndCs** [comment]
Thanks @wc-yaa
I hope you can do that, because I have no time and can't do everything myself. Currently I an teaching other languages to collage students and trying to learn zoom and such tools..
Thanks.

________________________________
From: wc-yaa ***@***.***>
Sent: Thursday, October 3, 2024 2:45 PM
To: VBAndCs/sVB-Small-Visual-Basic ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [VBAndCs/sVB-Small-Visual-Basic] A very suitable place! (Discussion #89)


https://litdev.uk/mybb/forumdisplay.php?fid=5

The above place is quite active. You can really show them how to solve each question in Small Visual Basic. Yes, show them the advantages of Small Visual Basic. In conclusion, it should be a nice place for the effective promotion of Small Visual Basic. Right?

**Reply by wc-yaa** [reply]
Me? Not convincing enough. What SVB lacks? Loyal and active users! Those people are loyal and active Small Basic users. If you(the author of SVB) sometimes(not always) appeared there to solve challenges, they will be more willing to download to try SVB. Furthermore, they will  be more willing to become loyal and actiive users of SVB as well. Loyal and active users is so important in the user community of SVB.  They    may be very helpful for promoting SVB. Sure.
